src/bin/clapper-app/clapper-app-uri-dialog.c

```diff
@@ -62,7 +62,7 @@ _open_uri_cb (AdwMessageDialog *dialog, GAsyncResult *result, GtkApplication *gt
 static void
 _read_text_cb (GdkClipboard *clipboard, GAsyncResult *result, GtkWidget *extra_child)
 {
-  GtkEntry *entry = GTK_ENTRY (extra_child);
+  AdwEntryRow *entry = ADW_ENTRY_ROW (extra_child);
```

Comment:
This one line can probably be removed completely and below casts just updated `GTK_EDITABLE (entry)` -> `GTK_EDITABLE (entry_row)`

Reply:
Done! I removed it